src/i18n/strings/en_EN.json

```
@@ -808,6 +808,12 @@
     "Please contact your homeserver administrator.": "Please contact your homeserver administrator.",
     "The person who invited you has already left.": "The person who invited you has already left.",
     "The person who invited you has already left, or their server is offline.": "The person who invited you has already left, or their server is offline.",
+    "This Room ID (%(id)s) cannot be joined. Try to join using a Room Alias (#room:example.com) instead.": "This Room ID (%(id)s) cannot be joined. Try to join using a Room Alias (#room:example.com) instead.",
+    "Why?": "Why?",
+    "A Room ID alone cannot be used to join a room unless your homeserver already participates in that room.": "A Room ID alone cannot be used to join a room unless your homeserver already participates in that room.",
+    "For a Room ID to be joinable over federation, a list of homeservers to attempt to join through must be provided. You didn't provide any.": "For a Room ID to be joinable over federation, a list of homeservers to attempt to join through must be provided. You didn't provide any.",
+    "The sole purpose of the domain at the end of a Room ID is to ensure it is unique.": "The sole purpose of the domain at the end of a Room ID is to ensure it is unique.",
+    "For these reasons, Room Aliases are the recommended way to join rooms.": "For these reasons, Room Aliases are the recommended way to join rooms.",
```

I'm not entirely sure how this needs to be formatted/escaped, but here goes:
```suggestion
    "In order to join a Room ID over federation, you need to explicitly provide a list of &quot;via&quot; homeservers which already know the room. One way to do so is using a <a href=\"https://github.com/matrix-org/matrix.to#optional-parameters\">matrix.to</a> URL. You can generate one in Element by right clicking on a room in your room list and selecting &quot;Copy room link&quot;.": "In order to join a Room ID over federation, you need to explicitly provide a list of &quot;via&quot; homeservers which already know the room. One way to do so is using a <a href=\"https://github.com/matrix-org/matrix.to#optional-parameters\">matrix.to</a>URL. You can generate one in Element by right clicking on a room in your room list and selecting &quot;Copy room link&quot;.",
    "For these reasons, Room Aliases are the recommended way to join rooms.": "For these reasons, Room Aliases are the recommended way to join rooms.",
```